src/components/button/button.style.js

```diff
@@ -31,6 +33,16 @@ export const StyledButtonSubtext = styled.span`
   display: block;
 `;
 
+function applyIconHeight({ theme, size, iconType }) {
+  if (isClassic(theme)) {
+    if (iconType === 'services') return '22px;';
+    return `${size === 'large' ? '20px' : '18px'};`;
+  }
+  if (iconType === 'services') return `${size === 'large' ? '28px' : '24px'}`;
```

Comment:
(A very minor comment!)

With this current architecture of `applyIconHeight()`, every `return` string needs to end with a `;`. On this line, the terminating `;` is missing in the `return` string.

The architecture would be more DRY if there was a single `;` at the end of this line where `applyIconHeight()` is called:
```
height: ${applyIconHeight}
```
and then the `return` strings in `applyIconHeight()` wouldn't need to end with `;`.

Reply:
Many thanks for refactoring this in 92f7fd8 :white_check_mark: